docs/source/index.rst

```diff
@@ -190,6 +190,7 @@ messy situations in everyday problems.
    phases-of-computation.rst
    remote-data-services.rst
    gpu.rst
+   contrib.rst
```

Comment:
Perhaps rename this file, since `contrib` sounds like contribution guidelines. `dask-contrib.rst`?